src/app/components/SubjectHeading/SubjectHeading.jsx

```diff
@@ -11,18 +13,25 @@ class SubjectHeading extends React.Component {
     const {
       subjectHeading,
       container,
+      sortBy,
     } = this.props;
     const {
       children,
+      range,
     } = subjectHeading;
     this.state = {
       open: !!children,
       narrower: (children || []),
+      sortBy: sortBy || "alphabetical",
```

Comment:
`sortType` or `sortAttribute` maybe?

Reply:
I'm using sortBy to match the api parameter as closely as possible, but I'd be open to changing both.

Reply:
I feel like `sortBy` works better for a query than a variable name. Not a big deal though